inst/stan/epinowcast.stan

```diff
@@ -125,8 +125,9 @@ transformed parameters{
 model {
   // priors for unobserved expected reported cases
   leobs_init ~ normal(eobs_init, 1);
+  eobs_lsd ~ normal(eobs_lsd_p[1], eobs_lsd_p[2]);
+  target += -normal_lccdf(0 | eobs_lsd_p[1], eobs_lsd_p[2]);
```

Comment:
Generally, I like vectorization not only because of efficiency but also because it makes the code more concise. In the present case, the two-liner is a bit unpleasant. It could be reduced to a one-liner as proposed by bbbales2 in the stan forum discussion:

`target += normal_lpdf(eobs_lsd | eobs_lsd_p[1], eobs_lsd_p[2]) - normal_lccdf(0 | eobs_lsd_p[1], eobs_lsd_p[2]) * g;`

This is not great but at least indicates that this is one likelihood component that belongs together. If we want to take it even further, we could write a user-defined function for the vectorized truncated normal distribution (until its available in stan itself). This would be the most readable version I guess!

Reply:
Yes, I agree. I like the idea of defining a custom vectorised truncated normal. Will add that in. Would help with mistakes as well (like missing all the multiplying by group...)